mintscript/src/tests/translate/snap.translate.tlock.transfer_01.txt

```diff
@@ -0,0 +1,26 @@
+WitnessScript(
+    Signature(
+        PublicKey(
+            PublicKey {
+                pub_key: Secp256k1Schnorr(
+                    Secp256k1PublicKey {
+                        pubkey_data: PublicKey(
+                            a3f77d24df8982b210a9506946739505f03e7c2ce4431116ea07e4069623fea3b846d320d74742e7618907d54d13ac2759b5e02843d627a804fdb0b492e272ef,
+                        ),
+                    },
+                ),
+            },
+        ),
+        Standard(
+            StandardInputSignature {
+                sighash_type: SigHashType(
+                    1,
+                ),
+                raw_signature: [
+                    81,
+                    81,
```

Comment:
Why does a single byte of 0x51 translate into two 81s, where 0x51=81?

Reply:
Note that this applies to all signatures.

Reply:
The function that creates the fake witness data drops (rather arbitrarily) two identical bytes there:
```rust
InputWitness::Standard(StandardInputSignature::new(sht, vec![byte; 2]))
```